pytri/js/ScatterLayer.js

```diff
@@ -1,20 +1,15 @@
 class ScatterLayer extends Layer {
     constructor(opts) {
         super(opts);
-        this.setData = this.setData.bind(this);
         this.radius = opts.radius || 0.15;
         this.colors = opts.colors || 0x00babe;
         if (typeof(this.colors) == "number") {
             this.c_array = false;
         } else {
             this.c_array = true;
         }
-        this.setData(opts.data);
-    }
-
-    setData(data) {
-        this.data = data;
-        this.requestInit;
+        this.data = opts.data;
```

Comment:
I trust that we didn't actually need that `setData` function or call here

Reply:
Was the `requestInit` call redundant?

Reply:
Yeah, the `setData` call was if we chose to add MORE data during runtime. Kinda useless for this layer in pytri